src/frontend/types/tracks.ts

```diff
@@ -45,7 +45,6 @@ export interface TimedText extends Resource {
   is_ready_to_play: boolean;
```

Comment:
There's a double `((` in the commit title

Reply:
:+1: